packages/alchemy/src/config/actions/getAccount.ts

```diff
@@ -15,7 +14,10 @@ export const getAccount = <TAccount extends SupportedAccountTypes>(
 ): GetAccountResult<TAccount> => {
   const accounts = config.clientStore.getState().accounts;
   if (!accounts) {
-    throw new ClientOnlyPropertyError("account");
```

Comment:
this throws in SSR without the SSR flag on, but I think if this gets called on the server it's fine to return a disconnected state